src/Features/CSharp/Portable/Completion/KeywordRecommenders/GlobalKeywordRecommender.cs

```diff
@@ -34,6 +34,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
             return
                 context.IsStatementContext ||
                 context.IsGlobalStatementContext ||
+                context.IsTypeContext ||
                 UsingKeywordRecommender.IsUsingDirectiveContext(context, forGlobalKeyword: true, cancellationToken) ||
                 context.IsAnyExpressionContext ||
                 context.IsObjectCreationTypeContext ||
```

Comment:
can likely remove IsIsOrAsTypeContext right?  And IsFunctionPointerTypeContext and IsObjectCreationTypeContext , right?

Reply:
Maybe, I'll check.

Reply:
Fixed.